src/main/webapp/js/angularApp/filters.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+/* global angular */
+
+/**
+ * Collection of filters used to massage $scope elements
+ * for display
+ *
+ * Just a shell for now.
+ * 
+ */
+
+angular.module('kanbanFilters', []).filter('blablabla', function () {
```

Comment:
Not using filters yet. Just a placeholder. I will make it clearer in the comment, thanks.